lib/errors.ts

```diff
@@ -157,6 +157,24 @@ export class BalenaReleaseNotFound extends BalenaError {
 }
 BalenaReleaseNotFound.prototype.code = 'BalenaReleaseNotFound';
 
+/**
+ * @summary Balena application membership role not found
+ * @class
+ * @public
+ *
+ * @param {(String|Number)} membershipRole - membership role name or id
```

Comment:
Just to be clear, I also mean to change this in all places like in here are well
```suggestion
 * @param {(String|Number)} applicationMembershipRole - application membership role name or id
```